egs/swbd/s5c/RESULTS

```diff
@@ -247,3 +247,7 @@ exit 0
 %WER 14.6 | 1831 21395 | 87.1 9.2 3.6 1.8 14.6 51.8 | exp/tri7uc-sat_mmi_b0.1/decode_eval2000_graph_sw1_tg_it4/score_17_0.0/eval2000.ctm.swbd.filt.sys
 # fisher 4gram rescoring,
 %WER 13.2 | 1831 21395 | 88.3 8.2 3.4 1.5 13.2 49.2 | exp/tri7uc-sat_mmi_b0.1/decode_eval2000_graph_sw1_fsh_fg_it4/score_19_0.0/eval2000.ctm.swbd.filt.sys
+
+# New nnet3 recipe with tdnn_7k
```

Comment:
in all the results, would it be possible to keep the order with tg/fg and other modifiers as the previous results, please?

Reply:
Hi @jtrmal, the result file has been updated.